include/bluetooth/hci.h

```diff
@@ -746,6 +748,10 @@ struct bt_hci_cp_le_set_adv_enable {
 #define BT_HCI_LE_SCAN_PASSIVE                  0x00
 #define BT_HCI_LE_SCAN_ACTIVE                   0x01
 
+/* Special scan address types for LL privacy */
+#define BT_HCI_LE_SCAN_RPA_OR_PUBLIC            0x02
```

Comment:
This sort of duplicates the info in lines 22 and 23? Is it necessary?

Reply:
They're not duplicates (besides in happening to have the same value). The scan parameter address type has different meaning than what's found in Enhanced Connection Complete or Advertising Report. I've tried to cover the different meaning in the define names.